### DISCUSSION TITLE
Help! how to make same height the card ?

### DISCUSSION THREAD

**Discussion by faruuhan** [nolimits4web/swiper#5896] [Q&A]
![image](https://user-images.githubusercontent.com/92959224/180001975-0a715e10-d428-4b9b-999f-815d52137733.png)

I struggle with this problem, and I am a newbie using swiper. can help me?

**Comment by Arhell** [comment]
Example please and everything can be done with css

**Reply by faruuhan** [reply]
> Example please and everything can be done with css

![image](https://user-images.githubusercontent.com/92959224/180200538-524fc18b-ce29-4e9e-a598-1bc14152c90a.png)
this example, i using tailwind for css

**Reply by Arhell** [reply]
Sorry, but there is an opportunity to make an example here
https://codesandbox.io/ or https://codepen.io/
